dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/DefaultTrackedEntityService.java

```diff
@@ -702,7 +702,7 @@ private boolean isProgramMinAttributesViolated(TrackedEntityQueryParams params)
 
   private void checkIfMaxTeiLimitIsReached(TrackedEntityQueryParams params, int maxTeiLimit) {
     if (maxTeiLimit > 0) {
-      int teCount = trackedEntityStore.getTrackedEntityCountForGridWithMaxTeiLimit(params);
+      int teCount = trackedEntityStore.getTrackedEntityCountForWithMaxTeiLimit(params);
```

Comment:
Not part of this PR but as we are here, can we rename `Tei` to something else in the method names and the variables?